ironfish-rust-nodejs/tests/demo.test.slow.ts

```diff
@@ -27,6 +27,12 @@ describe('Demonstrate the Sapling API', () => {
     expect(typeof key.spending_key).toBe('string')
   })
 
+  it('Should return phrase words from key input', () => {
+    const key = generateKey()
+    const words = wordsSpendingKey(key.spending_key, LanguageCode.English);
+    expect(words.split(' ').length).toEqual(24)
```

Comment:
you can actually hard code the result here, because it should never change if you want

Reply:
Tried it, `generateKey` actually generates a random key, so the value changes. Gonna leave as is